tests/e2escenarios/e2e_images_test.go

```diff
@@ -126,13 +126,13 @@ var _ = Describe("odo supported images e2e tests", func() {
 		})
 
 		It("Should be able to verify the openjdk-11-rhel8 image", func() {
-			oc.ImportImageFromRegistry("registry.redhat.io", filepath.Join("openjdk", "openjdk-11-rhel8:latest"), "java:8", project)
-			verifySupportedImage(filepath.Join("openjdk", "openjdk-11-rhel8:latest"), "openjdk", "java:8", project, appName, context)
+			oc.ImportImageFromRegistry("registry.redhat.io", filepath.Join("openjdk", "openjdk-11-rhel8:latest"), "java:8", "openjdk-11-rhel8")
```

Comment:
Shouldn't we assign `openjdk-11-rhel8` to project var on spec level

Reply:
It will be a unnecessary use of variable because the project name ```openjdk-11-rhel8``` has been created in the configuration level. Just need to use the namespace directly without assigning it to a new variable.